packages/composite-checkout/demo/index.js

```diff
@@ -24,28 +25,44 @@ const initialItems = [
 ];
 
 // These are used only for non-redirect payment methods
-/* eslint-disable no-console */
-const onSuccess = () => console.log( 'Payment succeeded!' );
-const onFailure = error => console.error( 'There was a problem with your payment', error );
-/* eslint-enable no-console */
+const onSuccess = () => window.alert( 'Payment succeeded!' );
+const onFailure = error => window.alert( 'There was a problem with your payment: ' + error );
```

Comment:
The behaviour right now is that we let people continue if there's an error in the form and then display an error in the end. In a future PR we should update the error handling so that people can't proceed from the payment method step if there are any inline errors.